node/execution/intrinsics/token/application/token_application.go

```diff
@@ -366,7 +366,7 @@ func (a *TokenApplication) ApplyTransitions(
 	}
 
 	wg := sync.WaitGroup{}
-	throttle := make(chan struct{}, runtime.GOMAXPROCS(-1))
+	throttle := make(chan struct{}, qruntime.WorkerCount(0, false))
```

Comment:
I've further lowered the concurrency of the mint transition applications to leave some cores available for networking even while the frame is built.

Reply:
validation isn't running on light provers

Reply:
Yes, this change was targeted at the beacon itself.